File: app/src/main/java/grupo3/rcmm/wifi_indoor_positioning_client/data/home/remote/APIInterface.java
package grupo3.rcmm.wifi_indoor_positioning_client.data.home.remote;

import grupo3.rcmm.wifi_indoor_positioning_client.data.home.model.Fingerprint;
import grupo3.rcmm.wifi_indoor_positioning_client.data.home.model.Location;
import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.POST;

public interface APIInterface {

    @POST("fingerprint")
    Call<Void> postFingerprint(@Body Fingerprint fingerprint);

    @POST("location")
    Call<Void> postLocation(@Body Location location);
}

File: app/src/main/java/grupo3/rcmm/wifi_indoor_positioning_client/data/home/remote/ApiInterface.kt
package grupo3.rcmm.wifi_indoor_positioning_client.data.home.remote

import grupo3.rcmm.wifi_indoor_positioning_client.data.home.model.Fingerprint
import retrofit2.Call
import retrofit2.http.Field
import retrofit2.http.FormUrlEncoded
import retrofit2.http.POST

interface ApiInterface {

    @POST("fingerprinting")
    @FormUrlEncoded
    fun postFingerprintings(@Field("info") lista: Call<List<Fingerprint>>)

    fun postLocation(@Field("pos") pos: Pair<Float, Float>)

}

